[2025-04-30T01:48:45.117+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_pipeline.transform_to_clean manual__2025-04-30T01:48:07.364762+00:00 [queued]>
[2025-04-30T01:48:45.140+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_pipeline.transform_to_clean manual__2025-04-30T01:48:07.364762+00:00 [queued]>
[2025-04-30T01:48:45.141+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-04-30T01:48:45.168+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): transform_to_clean> on 2025-04-30 01:48:07.364762+00:00
[2025-04-30T01:48:45.189+0000] {standard_task_runner.py:60} INFO - Started process 202 to run task
[2025-04-30T01:48:45.203+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'data_pipeline', 'transform_to_clean', 'manual__2025-04-30T01:48:07.364762+00:00', '--job-id', '23', '--raw', '--subdir', 'DAGS_FOLDER/data_pipeline_dag.py', '--cfg-path', '/tmp/tmpj7raoziy']
[2025-04-30T01:48:45.208+0000] {standard_task_runner.py:88} INFO - Job 23: Subtask transform_to_clean
[2025-04-30T01:48:45.260+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-04-30T01:48:45.391+0000] {task_command.py:423} INFO - Running <TaskInstance: data_pipeline.transform_to_clean manual__2025-04-30T01:48:07.364762+00:00 [running]> on host 59792d724231
[2025-04-30T01:48:45.707+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_pipeline' AIRFLOW_CTX_TASK_ID='transform_to_clean' AIRFLOW_CTX_EXECUTION_DATE='2025-04-30T01:48:07.364762+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-30T01:48:07.364762+00:00'
[2025-04-30T01:48:45.714+0000] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2025-04-30T01:48:45.715+0000] {base.py:83} INFO - Using connection ID 'postgres_default' for task execution.
[2025-04-30T01:49:23.426+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-04-30T01:49:23.450+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=data_pipeline, task_id=transform_to_clean, execution_date=20250430T014807, start_date=20250430T014845, end_date=20250430T014923
[2025-04-30T01:49:23.775+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-04-30T01:49:23.823+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
